src/docs/App.js

```diff
@@ -21,8 +21,9 @@ const Container = styled.main`
 `;
 
 const Content = styled.main`
-  display: flex;
-  flex: 1;
+  /* display: flex;
```

Comment:
we can omit this kind of comments (code's comments) and use git if we want to know the history of the file, it keeps project cleaner.